llama-index-core/llama_index/core/chat_engine/types.py

```diff
@@ -62,8 +62,8 @@ class StreamingAgentChatResponse:
     achat_stream: Optional[ChatResponseAsyncGen] = None
     source_nodes: List[NodeWithScore] = field(default_factory=list)
     _unformatted_response: str = ""
-    _queue: queue.Queue = field(default_factory=queue.Queue)
-    _aqueue: asyncio.Queue = field(default_factory=asyncio.Queue)
+    _queue: queue.Queue[str] = field(default_factory=queue.Queue)
+    _aqueue: asyncio.Queue[str] = field(default_factory=asyncio.Queue)
```

Comment:
I think this will cause an error on python3.8 (I remember either removing the type here or from another queue in the code)

Reply:
Ah fair. Pylance is yelling at me without it. You can't win.

Reply:
When python3.8 stops being supported, I will be the first to rip it out of the repo and improve the typing experience lol